homework/Repository.js

```diff
@@ -12,9 +12,38 @@ class Repository {
    * Render the repository info to the DOM.
    * @param {HTMLElement} container The container element in which to render the repository.
    */
-  render(container) {
+  render(leftDiv) {
     // TODO: replace the next line with your code.
-    Util.createAndAppend('pre', container, JSON.stringify(this.repository, null, 2));
+    const infoTable = Util.createAndAppend('table', leftDiv, { class: 'contributor-table' });
+    const tableBody = Util.createAndAppend('tbody', infoTable);
+    const rowName = Util.createAndAppend('tr', tableBody);
+    Util.createAndAppend('td', rowName, { text: 'Repository:', class: 'repo-name' });
+    const repoName = Util.createAndAppend('td', rowName);
+    Util.createAndAppend('a', repoName, {
+      target: '_blank',
+      href: this.html_url,
+      text: this.repository.name,
+    });
+    const rowDesc = Util.createAndAppend('tr', tableBody);
+    Util.createAndAppend('td', rowDesc, {
+      text: 'Description:',
+      class: 'repo-desc',
+    });
+    Util.createAndAppend('td', rowDesc, { text: `${this.repository.description}` });
+    const rowFork = Util.createAndAppend('tr', tableBody);
+    Util.createAndAppend('td', rowFork, {
+      text: 'Forks:',
+      class: 'repo-Fork',
+    });
+    Util.createAndAppend('td', rowFork, { text: `${this.repository.forks}` });
+    const rowUpDate = Util.createAndAppend('tr', tableBody);
+    Util.createAndAppend('td', rowUpDate, {
+      text: 'Updated:',
+      class: 'repo-update',
+    });
+    Util.createAndAppend('td', rowUpDate, {
+      text: `${new Date(this.repository.updated_at).toDateString()}`,
+    });
   }
 
   /**
```

Comment:
In line 52 (not shown here), you are still using the `fetchJSON()` function. This is inconsistent with what you did in the `App` class. Can you modify this to also use `fetch`?

Reply:
oh yesss, i found it and replace it with fetch